Snake/BusinessLayer/FoodFolder/Food.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace BusinessLayer.FoodFolder
+{
+    public class Food
+    {
+        public int yLocation { get; set; }
+        public int xLocation { get; set; }
+        public int pointsWorth { get; set; }
```

Comment:
Classes that inherit from this class set their own values. I tried setting it to private, and the BasicFood and AdvancedFood classes were not too happy.